Lib/test/test_venv.py

```diff
@@ -24,8 +24,10 @@
 except ImportError:
     ctypes = None
 
-skipInVenv = unittest.skipIf(sys.prefix != sys.base_prefix,
-                             'Test not appropriate in a venv')
+requireVenvCreate = unittest.skipUnless(
+    sys.platform.startswith('win')
```

Comment:
Can you please add a comment explaining why tests don't need to be skipped on Windows? Is it a new feature of Python 3.8? Is it the new "wrapper" that you merged recently? Maybe add a link to the bpo which added this wrapper.